sp/src/game/server/hl2/npc_combine.cpp

```diff
@@ -5777,6 +5862,20 @@ bool CNPC_Combine::PickTacticalLookTarget( AILookTargetArgs_t *pArgs )
 	}
 }
 
+//-----------------------------------------------------------------------------
+// Blixibon - Hack to kick in the above look target code as soon as we're in combat
+//-----------------------------------------------------------------------------
+void CNPC_Combine::OnStateChange( NPC_STATE OldState, NPC_STATE NewState )
```

Comment:
This seems unrelated to the purpose of the PR - have you tested this thoroughly? I don't really have any objections, I just want to double check it